_maps/RandomRuins/LavaRuins/lavaland_biodome_beach.dmm

```diff
@@ -985,10 +985,10 @@
 /turf/open/floor/wood,
 /area/ruin/powered/beach)
 "GR" = (
-/obj/item/toy/beach_ball/holoball/dodgeball,
-/obj/item/toy/beach_ball/holoball/dodgeball,
-/obj/item/toy/beach_ball/holoball/dodgeball,
-/obj/item/toy/beach_ball/holoball/dodgeball,
+/obj/item/toy/dodgeball,
+/obj/item/toy/dodgeball,
+/obj/item/toy/dodgeball,
+/obj/item/toy/dodgeball,
```

Comment:
Please add this to your update paths script

Reply:
Applied with https://github.com/tgstation/tgstation/pull/72459/commits/22a98beca584b54b10c0cfa7a9eda51f314505be